libs/contrib/System/Path.idr

```diff
@@ -352,115 +397,147 @@ export
 joinPath : List String -> String
 joinPath xs = foldl (</>) "" xs
 
-||| Returns the parent and child.
+||| Split path into components.
+|||
+||| ```idris example
+||| splitPath "/usr/local/etc" == ["/", "usr", "local", "etc"]
+||| splitPath "tmp/Foo.idr" == ["tmp", "Foo.idr"]
+||| ```
+export
+splitPath : String -> List String
+splitPath path = map show $ splitPath' (parse path)
+
+||| Split the path into parent and child.
 |||
 ||| ```idris example
 ||| splitParent "/usr/local/etc" == Just ("/usr/local", "etc")
 ||| ```
 export
 splitParent : String -> Maybe (String, String)
-splitParent p = do (a, b) <- splitParent' (parse p)
-                   pure $ (show a, show b)
+splitParent path =
+  do
+    (parent, child) <- splitParent' (parse path)
+    pure (show parent, show child)
 
-||| Returns the path without its final component, if there is one.
+||| Return the path without its final component, if there is one.
 |||
-||| Returns Nothing if the path terminates in a root or volume.
+||| Returns Nothing if the path terminates by a root or volume.
```

Comment:
```suggestion
||| Return Nothing if the path terminates by a root or volume.
```

Its using the imperative tense just above :)

Reply:
This is adopted from Rust documents. https://doc.rust-lang.org/std/path/struct.Path.html#method.is_relative

I think it is an abbr for "*It* returns ..."

Reply:
Yes, but in the rest of this document, you replace present tense by imperative tense by removing the `s` on lots of verbs. The change is fine, but it should be consistent, line 422 does this change by replacing `Returns` by `Return`, so the change should be carried over to this line too.